tests/lib/prepare.sh

```diff
@@ -895,15 +892,19 @@ setup_reflash_magic() {
         # FIXME: install would be better but we don't have dpkg on
         #        the image
         # unpack our freshly build snapd into the new snapd snap
+        # shellcheck disable=SC2031
```

Comment:
I see a lot of new disable=SC2031, are all of them needed?
I don't see vars modifications on some of them

Reply:
shellcheck is apparently confused by the subshell, but since I moved the helper to separate file, the subshell is no longer needed